shell/app/modules/project/pages/apps/app-form.tsx

```diff
@@ -133,6 +136,7 @@ const CreationForm = () => {
       getComp: ({ form }: { form: FormInstance }) => (
         <AppTypeSelect
           imgOptions={useOption}
+          canCreateMobileApp={currentProjectClusters?.length > 0}
```

Comment:
Add disabled property in `imgOptions` instead of pass a particular property.
Why AppTypeSelect have to know mobile app ? It just render a app type list.